app/gui/view/graph-editor/src/new_node_position.rs

```diff
@@ -29,6 +29,12 @@ pub mod free_place_finder;
 /// === New Node Positioning ===
 /// ============================
 
+/// The maximum distance to the nearest grid line in Magnet Alignment algorithm.
+///
+/// This value defines the "sensitivity" of the Magnet Alignment for new nodes - greater values
+/// lead to more "aggressive" snapping.
+const MAGNET_ALIGNMENT_THRESHOLD: f32 = 150.0;
```

Comment:
Looking at the videos I feel that this area is too big now. But to know exactly we need to test it live. Just saying, nothing to be fixed here for now.

Reply:
Yeah, totally, this requires some additional testing.